TASK-2_2/Currency.cs

```diff
@@ -0,0 +1,11 @@
+namespace Task2_2;
+
+public class Currency
```

Comment:
Just enum Currency, without inner enum in class Currency.

Reply:
Ok, thanks!